src/pages/hegel/guides/presuppositionless-thinking.md

```diff
@@ -0,0 +1,31 @@
+
+## How does presuppositionlessness work?
+
+Øet us briefly outline how exactly presuppositionlessness works in practice. We have already stated that presuppositionlessness does not require us to do away with language or our reservoir of concepts. We are not trying to reinvent the wheel. Rather, we are *not* taking for granted that our concepts are true. What we are inquire into is whether we have understood the nature of our concepts correctly or not. In this sense, then, we are not calling into question the conditions that enable us to do philosophy in the first place. Our presuppositionlessness is, instead, a methodological presuppositionlessness.
+
+The first tenet of methodological presuppositionlessness is “do not take anything for granted”. As such, whilst doing philosophy we will never be driven by what philosophers have normally taken ‘x’ to be, or by what we think ‘y’ is based on experience. We will set aside our assumptions about how philosophical concepts ought to be. In the absence of any assumptions about how philosophical assumptions ought to be, how is our philosophical inquiry to proceed? This brings us to the matter of the beginning of philosophy, which is a whole topic unto itself; for now, we will assume that there is nothing problematic about the beginning. Hegel’s strategy for beginning his philosophical inquiry, despite setting aside all assumptions, is to abstract all content from thought and to see where thought takes us. This is the second methodological tenet: “attend to the immanent determination of thought and do not import anything external into it”. Once we have set aside all of our assumptions and abstracted all content from thought, our philosophical inquiry will proceed by attending to the immanent character of our thought. Whatever that may be. As it turns out, when we abstract all content from thought what we are left with is the simple is-ness of thought or “pure being”. If we think about “pure being” and consider its characteristics we will discover that it is, in fact, “pure nothing”. And so on and so forth, the dialectic of thought proceeds. 
```

Comment:
Not sure about this determining of "tenets", even though I like the idea a lot, I worry that we start ossifying presuppositionless thinking - creating presuppositions for the very thing itself.

Reply:
Typo at the start of line 29 has been fixed.

Reply:
Perhaps this isn't the best place to put it. But I do think that that presuppositionless thinking is a kind of a methodological tenet rather than a blanket commitment, precisely because there are things that Hegel is presuppositionless about and things that he is not presuppositionless about. And that distinction requires a justification. In this sense, I do think that presuppositionless thought is at risk of being itself the product of a presupposition without a proper justification for itself. 
But this is something that, perhaps, should not be included in this post.

Reply:
I agree that there is method to presuppositionless thinking; it is a distinct way of thinking about something. But tenet sounds very much like a "rule-bound" thing and I'm not sure if we should be introducing any kind of language of rules at this stage. Maybe I am over-reacting? 

On the other hand, "tenet" has some very nice connotations to it, initially meaning "to hold", so that if you are committed to presuppositionless thinking you are thereby committed "to hold" certain notions. And so these notions that you "hold" are not so much "rules" from the top-down as conclusions from the bottom-up from the fact you are thinking presuppositionlessly. Does this make sensee?

Reply:
That does make sense. I've never thought about it as whether it's motivated from bottom-up or top-down. 

These are my concerns with presuppositionless thought: (1) There is no immanent justification for it; (2) Since there is no immanent justification, the justifications are necessarily external to the subject-matter that is being treated presuppositionlessly; (2a) this does not mean that the subject-matter that develops presuppositionlessly is not immanent, but only that the decision to be presuppositionless is not immanent but external; (3) Presuppositionless thought is based on a number of possible assumptions, freedom, Kant's dialectic, etc; (4) whichever assumption one espouses, there is a further assumption about which things one must be presuppositionless about (how we comprehend our concepts) and things which we need be presuppositionless about (whether our experience is true or whether language is an apt medium for doing philosophy) and I have some other thoughts about just how complex a requirement presuppositionless thought is; (5) and it is this final point (point 4) that requires that one constantly pays attention to what they're doing. 

It is because presuppositionless thought, conceived as the above, requires one to be meticulous about what they are and what they are not presuppositionless about, that I thought of using the word "tenet". If you think tenet is too loaded a term, and you agree with the above (broadly), then I'm open to alternative terms.

Reply:
Great considerations. Very valuable and would be great material for a more advanced essay on this topic. 

> (2a) this does not mean that the subject-matter that develops presuppositionlessly is not immanent, but only that the decision to be presuppositionless is not immanent but external;

Not sure we need to pivot to externality here. The "decision" to be presuppositionless could be immanent, but not apparent or explicit. That is, presuppositionless thinking does not need to rely on any externalities for its determination (except enabling conditions) - it is self-determining. But the full scope or comprehension of it isn't made explicit. So, the moment one is engaged in thinking presuppositionlessly, no more and no less is needed; there isn't anything it's "based on". We can extract aspects or "tenets" out of it but these are after the fact. 

Yes, justification for it seems difficult since even the term "justification" is at the zero level "unjustified" or not yet developed. But we don't enter into that kind of raging, unshackled skepticism (perhaps another good topic to explore: why doesn't presuppositionless thinking devolve into manic skepticisim?). 

Regardless, we can pin these issues. Should we merge these changes in? (Open the PR up for review if you agree.)

Reply:
I'm not sure I agree that we extract tenets after the fact of thinking presuppositionlessly. But I think the best way to further illustrate the point would be to go through a logical determination together. 

In any case, I agree that we can pin these issues and open up different threads for them.